config/variables.cfg

```diff
@@ -14,9 +14,9 @@ if ! [ -x "$(command -v jq)" ]; then sudo apt update; sudo apt install jq -qq -y
 #-------------------------------#### ! CUSTOMIZE THESE ! ####-------------------------------#
 
 # ENVIRONMENT lets the script know where it will be run. Possible options are:
-#        - mainnet --> will use configs for the main Elrond Network                           
-#        - testnet --> uses configs for the Elrond Testnet Network
-#        - devnet --> allows access to the long-lived Elrond Devnet Network
+#        - mainnet --> will use configs for the main MultiversX Network                           
+#        - testnet --> uses configs for the MultiversX Testnet Network
```

Comment:
drop `Network`? `Testnet` already indicates a network. Also for devnet

Reply:
Solved !